src/main/java/com/chopshop166/chopshoplib/outputs/SwPIDSpeedController.java

```diff
@@ -133,90 +100,24 @@ public PIDController getController() {
 
     /** Enable the PID controller. */
     public void enablePID() {
-        if (!enabled) {
-            enabled = true;
+        if (!pidEnabled) {
+            pidEnabled = true;
             dog.enable();
         }
     }
 
     /** Disable the PID controller. */
     public void disablePID() {
-        enabled = false;
+        pidEnabled = false;
         dog.disable();
     }
 
-    @Override
-    public void initSendable(final SendableBuilder builder) {
-        motor.initSendable(builder);
-    }
-
-    @Override
-    public void setP(final double kp) {
-        pid.setP(kp);
-    }
-
-    @Override
-    public void setI(final double ki) {
-        pid.setI(ki);
-    }
-
-    @Override
-    public void setD(final double kd) {
-        pid.setD(kd);
-    }
-
-    @Override
-    public void setF(final double kf) {
-        feedforward = kf;
-    }
-
     @Override
     public void setSetpoint(final double setPoint) {
         pid.setSetpoint(setPoint);
         this.setpoint = setPoint;
     }
 
-    @Override
-    public void set(final double speed) {
-        motor.set(speed);
-    }
-
-    @Override
-    public double get() {
-        return motor.get();
-    }
-
-    @Override
-    public void setInverted(final boolean isInverted) {
-        motor.setInverted(isInverted);
-    }
-
-    @Override
-    public boolean getInverted() {
-        return motor.getInverted();
-    }
-
-    @Override
-    public void disable() {
-        motor.disable();
-    }
-
-    @Override
-    public void stopMotor() {
-        motor.stopMotor();
-    }
-
-    @SuppressWarnings("removal")
-    @Override
-    public void pidWrite(final double output) {
-        motor.pidWrite(output);
-    }
-
-    @Override
-    public IEncoder getEncoder() {
-        return motor.getEncoder();
-    }
-
     /** Calculate the PID value, and set the speed controler to the result. */
     private void calculate() {
```

Comment:
This is aliasing `SmartMotorController.calculate()` isn't it?

Reply:
Overloading - different argument types and counts. I agree that the names could be better though - naming is hard.